zubhub_frontend/zubhub/src/views/profile/Profile.jsx

```diff
@@ -282,34 +315,70 @@ function Profile(props) {
                         className={classes.moreInfoStyle}
                         component="h5"
                       >
-                        {profile.members_count} {t('profile.membersCount')}
+                        <box className={classes.moreInfoTitle}>
+                         {t('profile.membersCount')} 
+                        </box>
+                        <box className={classes.moreInfoCount}>
+                         {profile.members_count}
+                        </box>
                       </Typography>
                     </Link>
                   ) : null}
-                </Box>
+                </Box> 
               </Box>
             </Container>
           </Paper>
 
           <Container maxWidth="md">
-            <Paper className={classes.profileLowerStyle}>
-              <Typography
-                gutterBottom
-                component="h2"
-                variant="h6"
-                color="textPrimary"
-                className={classes.titleStyle}
-              >
-                {!profile.members_count
-                  ? t('profile.about.label1')
-                  : t('profile.about.label2')}
-              </Typography>
-              {profile.bio
-                ? profile.bio
-                : !profile.members_count
-                ? t('profile.about.placeholder1')
-                : t('profile.about.placeholder2')}
-            </Paper>
+            <Box className={classes.aboutMeBadgeBox}>
+              <Paper className={classes.profileLowerStyle}>
+                <Typography
+                  gutterBottom
+                  component="h2"
+                  variant="h6"
+                  color="textPrimary"
+                  className={classes.titleStyle}
+                >
+                  {!profile.members_count
+                    ? t('profile.about.label1')
+                    : t('profile.about.label2')}
+                </Typography>
+                {profile.bio
+                  ? profile.bio
+                  : !profile.members_count
+                  ? t('profile.about.placeholder1')
+                  : t('profile.about.placeholder2')}
```

Comment:
FYI, it doesn't apply to PR but it  would be easy to understand what the translation might be if we add context in the key.
For example if we add `profile.about.memberPlaceholder` and `profile.about.groupPlaceholder` instead of placeholder1 and placeholder2 it would be easy to know what kind of message will be displayed

Reply:
this was the original translation. Even I was confused about it in the beginning. 
yeah, I'll update that too.

Reply:
thankyou so much @kamthamc for all of your comments, I'll definitely work on them and update you soon.🙂